Source/JavaScriptCore/inspector/protocol/Page.json

```diff
@@ -24,6 +24,21 @@
                 "WebSecurityEnabled"
             ]
         },
+        {
+            "id": "UserPreference",
+            "type": "string",
+            "description": "List of user preferences that WebInspector can override.",
+            "enum": [
+                "PrefersReducedMotion",
+                "PrefersContrast"
+            ]
+        },
+        {
+            "id": "UserPreferenceValue",
+            "type": "string",
+            "enum": ["System", "On", "Off"],
```

Comment:
I don't think we need an explicit `"System"` value since the use of this is already `"optional": true`.  The "revert to system" should be (and looking at the code it seems like it already is) just not providing the `value`.

Also, I don't think `"On"` and `"Off"` are really the right values for this.  I think was should just make the `value` into a `"type": "string"` since each one of these user preferences (and the others we should add in the future) could have arbitrary values that are not `"On"` and `"Off"`.  Each `UserPreference` should just do it's own parsing in the backend to figure out what each possible string value is allowed.